hibernate-core/src/main/java/org/hibernate/boot/jaxb/internal/AbstractBinder.java

```diff
@@ -36,15 +38,18 @@ public abstract class AbstractBinder<T> implements Binder<T> {
 
 	private final LocalXmlResourceResolver xmlResourceResolver;
 
+	protected InputStreamAccess streamAccess;
+
 	protected AbstractBinder(ResourceStreamLocator resourceStreamLocator) {
 		this.xmlResourceResolver = new LocalXmlResourceResolver( resourceStreamLocator );
 	}
 
-	public abstract boolean isValidationEnabled();
+	public abstract XmlValidationMode getXmlValidationMode();
```

Comment:
Unless I am mistaken we don't need this method.  Each of the 2 impls can just deal with it locally.